src/edu/ucsd/library/xdre/statistic/analyzer/LogAnalyzer.java

```diff
@@ -90,11 +90,17 @@ public void analyze(File logFile) throws IOException{
 					if (line.indexOf("\"-\"", spIdx) > 0 || line.indexOf("archive.org_bot", spIdx) > 0)
 						continue;
 
-					// ignore hits from MSCL collection pages: http://library.ucsd.edu/speccoll/
-					if (line.indexOf("http://library.ucsd.edu/speccoll/", spIdx) > 0)
-						continue;
-					
 					uri = getUri(line);
+					// Hits from MSCL collection pages: http://library.ucsd.edu/speccoll/
```

Comment:
@mcritchlow Do you mean the original solution in https://github.com/ucsdlib/damsmanager/pull/131/commits/17630d058a6f974ce854f306ab841c2603e193d7 to separate the counts for "mscl", and then add to DAMS stats sounds better?